sep_021.md

```diff
@@ -94,7 +95,13 @@ Examples <a name='examples'></a>
 
 ### 3.1 Flow Cytometry <a name="example1"></a>
 
-**Figure 3:** An example of using the classes from this 
+**Figure 3:** An example of using the classes from this
+
+### 3.2 Interlab Example <a name="example2"></a>
+
+**Figure 4:** Interlab Example
+
+![ExperimentalData class UML diagram](images/sep_021_interlab_example.png "ExperimentalData class UML diagram")
```

Comment:
I believe this diagram should have a wasGeneratedBy linking from the ExperimentalData to the Implementation as well.  The reasoning will be made clearer if you show two samples, not just one.

Reply:
Two questions:

- When you say two samples, do you mean two Implementations, both of which would be connected to the Experiment through wasDerivedFrom links?
- Should it be part of SEP21 that wasGeneratedBy links should be made from ExperimentalData to 'participating' Implementations?

Reply:
1) Yes, two implementations.  In fact, I think it would be even better with 3, e.g., "positive control", "negative control," and "study sample".

2) It was implicit before (since SEP21 doesn't change the fact that you would have wanted to make those links if Experiment didn't exist), so we might as well make it explicit.